src/Widgets/Switchboard/Page.vala

```diff
@@ -0,0 +1,23 @@
+/*
+* Copyright (c) 2017 elementary LLC. (https://elementary.io)
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation, either version 2.1 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Library General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public
+* License along with this program; if not, write to the
+* Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+* Boston, MA 02110-1301 USA
+*/
+
+public class Switchboard.Page : Gtk.ScrolledWindow {
```

Comment:
I think what should be done in this case is that this class should be an `abstract` class and only contain needed properties. Remember that if this gets into libswitchboard / Granite, you could provide another class named e.g: `SimplePage` and implement a common implementation of a page.